enaml/testing/utils.py

```diff
@@ -0,0 +1,163 @@
+#------------------------------------------------------------------------------
+# Copyright (c) 2013-2017, Nucleic Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#------------------------------------------------------------------------------
+"""Generic utility functions for testing.
+
+"""
+from time import sleep
+from contextlib import contextmanager
+
+import enaml
+from ..application import timed_call
+from ..widgets.api import Window, Dialog
+with enaml.imports():
+    from ..stdlib.message_box import MessageBox
+
+
+def get_window(app, cls=Window):
+    """Convenience function running the event loop and returning the first
+    window found in the set of active windows.
+
+    Parameters
+    ----------
+    cls : type, optional
+        Type of the window which should be returned.
+
+    """
+    sleep(0.1)
+    app.process_events()
+    w_ = None
+    for w in Window.windows:
+        if isinstance(w, cls):
+            w_ = w
```

Comment:
you can just `return w` here instead of using the temp variable.

Reply:
Will update